src/rgw/rgw_admin.cc

```diff
@@ -489,6 +489,9 @@ void usage()
   cout << "\nradoslist options:\n";
   cout << "   --rgw-obj-fs                  the field separator that will separate the rados object name from the rgw object name;\n";
   cout << "                                 additionally rados objects for incomplete multipart uploads will not be output\n";
+  cout << "\nlist object in Bucket options:\n";
+  cout << "   --max-entries                 the number of objects will show one time, default 1000, no more than 10000\n";
+  cout << "   --marker                      the marker for listing object in bucket\n";
```

Comment:
I'd suggest "the marker used to specify on which entry the listing begins, default none (i.e., very first entry)".